packages/core/src/components/ModalNew/Modal/Modal.module.scss

```diff
@@ -5,6 +5,10 @@
 }
 
 .modal {
+  --top-actions-spacing: var(--spacing-large);
+  --top-actions-width: var(--spacing-xl);
+  --modal-inline-padding: var(--spacing-xl);
```

Comment:
Are these changeable between themes?

Reply:
shouldn't be

Reply:
So I'm curious why the variables 😅

Reply:
to be used internally by the other child components of the Modal
I can put it in context but it is an overkill

Reply:
to be more specific and declarative
this:
```
padding-inline-end: calc(var(--top-actions-spacing) + var(--top-actions-width) + var(--spacing-medium));
```
looks much clearer than:
```
padding-inline-end: calc(var(--spacing-large) + var(--spacing-xl) + var(--spacing-medium));  
```

also, it is being used in multiple places, so if one value changes it would automatically match for the other places (design also wants to change the spacings and width on the new upcoming full-view modal, so it would even be more usable)